.jenkins/continuous.groovy

```diff
@@ -357,6 +60,7 @@ pipeline {
                                     -D ARBORX_ENABLE_EXAMPLES=ON \
                                     -D ARBORX_ENABLE_BENCHMARKS=ON \
                                     -D ARBORX_ENABLE_ONEDPL=ON \
+                                    -D ONEDPL_PAR_BACKEND=serial \
```

Comment:
Guess this is fine

Reply:
I don't like it at all that we have to provide this when configuring `ArborX` and downstream projects and can't do it when configuring and building `oneDPL` but I don't think there is much we can do about it.

Reply:
I don't understand what this does.  Is it something that influences the `oneDPL` package that is found?

Reply:
Yes, see https://github.com/arborx/ArborX/pull/614#issuecomment-1009234281.

Reply:
OK I found https://github.com/oneapi-src/oneDPL/blob/4f9761145b89fe0107e61cb7eeae42145b74aadb/cmake/README.md#using-onedpl-package

Why is it something that should be specified by the user at configuration time and not something that we can deal with ourselves
```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 313a262..8416432 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,6 +30,9 @@ endif()

 cmake_dependent_option(ARBORX_ENABLE_ONEDPL "Enable oneDPL support" ON "Kokkos_ENABLE_SYCL" OFF)
 if(Kokkos_ENABLE_SYCL AND ARBORX_ENABLE_ONEDPL)
+  if(NOT ONEDPL_PAR_BACKEND)
+    set(ONEDPL_PAR_BACKEND serial)
+  endif()
   find_package(oneDPL REQUIRED)
   target_link_libraries(ArborX INTERFACE oneDPL)
 endif()

Reply:
I tried to restrict other uses of `oneDPL` as little as possible so that the user has full control. Normally, i.e., when we have a full `oneAPI` framework available the default should also not provide any problems since we have `OpenMP` support available.
On the other hand, it would be nicer if the user doesn't have to fight with the fact that the `oneDPL` developers didn't take into account that `OpenMP` might not be available, especially given that we have to reconfigure `oneDPL` in downstream projects.

Reply:
For reference, at the dev meeting we brought out that we would need to conditionally set the variable in our exported configuration before the `find_dependency(oneDPL)` and we decided to abstain for now.